### PR TITLE
Minor logo documentation updates and language version implementation

### PR DESCRIPTION
**Pull request by PetriLahdelma**
Minor logo documentation updates and russian logo implementation

## Description
Documentation updates and russian logo implementation

## Related Issue

Closes #

## How Has This Been Tested?
running docisite and stories locally

## Screenshots (if appropriate):
Russian logo seems to work also with footer and navigation
Näyttökuva 2021-11-16 kello 10 30 35
Näyttökuva 2021-11-16 kello 10 35 28

### CODE REVIEW

**Review by laurakarhu — APPROVED** [review]
Should be alright now. :)

**Review by minevala — COMMENTED** [review]
Nice work 👍 
I noticed minor formatting issues with the SVG XMLs. They should be easy to fix.

File: site/static/assets/logo/fi/m/helsinki-fi-m-black.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+        </g>
+    </g>
+</svg>
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply site/static/assets/logo/fi/m/helsinki-fi-m-black.svg:10]: I will try to fix this.

[the remark this answers is not in the document]
Comment (ronijaakkola) [reply site/static/assets/logo/fi/m/helsinki-fi-m-black.svg]: Looking better now?

[the remark this answers is not in the document]
Comment (minevala) [reply site/static/assets/logo/fi/m/helsinki-fi-m-black.svg]: Yes 👍 
Git throws warnings of missing newlines, at least in terminal

**Review by minevala — APPROVED** [review]
🏅

**Review by ronijaakkola — APPROVED** [review]

**Pull request merged**